ingestion/functions/parsing/brazil_espirito_santo/espirito_santo.py

```diff
@@ -0,0 +1,326 @@
+    items = (serological_method1, serological_method2, serological_method3)
+    if pcr:
+        return "PCR test"
+    elif any(item not in _NONE_TYPES for item in items):
+        return "Serological test"
+    elif "Clinico" in test_class:
+        return "Clinical diagnosis"
+
+
+def convert_preexisting_conditions(lung: str, kidney: str, diabetes: str,
+                                   cardiovascular: str, obesity: str):
+    preexistingConditions = {}
+    items = (lung, kidney, diabetes, cardiovascular, obesity)
+    if all(item in _NONE_TYPES for item in items):
+        return None
+
+    preexistingConditions["hasPreexistingConditions"] = True
+    comorbidities = []
+
+    if lung == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadePulmao"])
+    if kidney == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeRenal"])
+    if diabetes == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeDiabetes"])
+    if cardiovascular == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeCardio"])
+    if obesity == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeObesidade"])
+
+    if comorbidities:
+        preexistingConditions["values"] = comorbidities
+
+    return preexistingConditions
+
+
+def convert_symptoms(headache: str, throat: str, fever: str, cough: str, diarrhoea: str, dyspnea: str):
+    symptoms = {}
+    items = (headache, throat, fever, cough, diarrhoea, dyspnea)
+    if all(item in _NONE_TYPES for item in items):
+        return None
+
+    symptoms["status"] = "Symptomatic"
+    values = []
+
+    if headache == "Sim":
+        values.append(_SYMPTOMS_MAP["Cefaleia"])
+    if throat == "Sim":
+        values.append(_SYMPTOMS_MAP["DorGarganta"])
+    if fever == "Sim":
+        values.append(_SYMPTOMS_MAP["Febre"])
+    if cough == "Sim":
+        values.append(_SYMPTOMS_MAP["Tosse"])
+    if diarrhoea == "Sim":
+        values.append(_SYMPTOMS_MAP["Diarreia"])
+    if dyspnea == "Sim":
+        values.append(_SYMPTOMS_MAP["DificuldadeRespiratoria"])
+
+    if values:
+        symptoms["values"] = values
+
+    return symptoms
+
+
+def convert_ethnicity(ethnicity: str):
+    if ethnicity == "Preta":
+        return "Black"
+    elif ethnicity == "Parda":
+        return "Mixed"
+    elif ethnicity == "Amarela":
+        return "Asian"
+    elif ethnicity == "Branca":
+        return "White"
+    elif ethnicity == "Indigena":
+        return "Indigenous"
+
+
+def convert_demographics(gender: str, age: str, ethnicity: str, healthcare_proffesional: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        if age == "90 anos ou mais":
+            demo["ageRange"] = {"start": 90, "end": 120}
+        else:
+            # Age in format '20 a 29 anos', except for '05 a 9 anos' which has a leading 0
+            age_range = age.partition(" a ")
+            if age_range[0] == "05":
```

Comment:
Is this extra case necessary? I believe `float("05")` should return `5.0`. Python tends to automatically handle leading zeroes - this is quite annoying in the case of zip codes, but useful here.

Reply:
I agree with @calremmel here. Write a unit test for the age parsing, then remove this code and check whether the test still passes :).

Reply:
So initially I didn't have a specific case for the 05 and it did give me an error like "Python cannot deal with leading 0" when I was doing unit testing, hence why I added the case... Will try and reproduce and post here

Reply:
It looks as though there is a difference between Python 2 and Python in whether they accept leading 0's https://medium.com/techmacademy/leading-zeros-in-python-a-hidden-mystery-revealed-ee3e3065634d. Anyway will try and reproduce error, may shed some light

Reply:
```
>>> float("05 a 9 anos".partition(" a ")[0])
5.0
```
should be OK

Reply:
@AnyaLindstromBattle ah yeah, there's a convention from C that integers with a leading 0 are treated as base 8 (the reason has something to do with a specific model of computers from the 1970s that isn't very exciting). Converting '09' into an int would fail with python 2, but not 3 (and this is all moot for the float type).